src/autocomplete.js

```diff
@@ -224,6 +225,10 @@ export default class Autocomplete extends Component {
     }
   }
 
+  handleInputClick (evt) {
+    this.handleInputChange(evt)
```

Comment:
To solve the fact that this doesn't work with keyboard use, what about triggering this in `handleInputFocus`?